akka-actor-typed-tests/src/test/scala/docs/akka/typed/SpawnProtocolDocSpec.scala

```diff
@@ -64,7 +64,7 @@ class SpawnProtocolDocSpec extends ScalaTestWithActorTestKit with WordSpecLike {
         system.ask(SpawnProtocol.Spawn(behavior = HelloWorld.greeter, name = "greeter", props = Props.empty))
 
       val greetedBehavior = Behaviors.receive[HelloWorld.Greeted] { (context, message) =>
-        context.log.info("Greeting for {} from {}", message.whom, message.from)
+        context.log.info("Greeting for {} from {}", Array(message.whom, message.from))
```

Comment:
didn't it work with plain parameters (varargs)?

Reply:
I got this error:

```ambiguous reference to overloaded definition,
[error] both method info in trait Logger of type (x$1: String, x$2: Object*)Unit
[error] and  method info in trait Logger of type (x$1: String, x$2: Any, x$3: Any)Unit```

so now I changed it to:
``` context.log.info("Greeting for {} from {}", Array(message.whom, message.from):_*)```


sorry 
``` context.log.info("Greeting for {} from {}", Array(message.whom, message.from))``` was wrong